src/java/fr/paris/lutece/plugins/oauth2/dataclient/AbstractDataClient.java

```diff
@@ -260,6 +279,33 @@ public String getData( Token token )
         return strResponse;
     }
 
+    /**
+     * Parse JWT response
+     * @param strCompactJWT
+     * @return string
+     */
+    private String parseJWTResponse( String strCompactJWT )
+    {     
+        try
+        {
+            Claims claims =_jWTParser.parseJWT( strCompactJWT, _authClientConf, _authServerConf, Claims.class, _logger );
```

Comment:
I'm not sure this would work when using any other parser than JjwtParser, because other parsers do not return the appropriate class or kown how to convert from their internal classes to JJwt ones